dg_test.go

```diff
@@ -160,13 +159,11 @@ func testSingleResolutionDependency(
 	assert.NoError(t, d.PushStartingNodes())
 	readyNodes := d.PopReadyNodes()
 	assert.Equals(t, len(readyNodes), 1)
-	assert.Equals(t, readyNodes[0].ID(), dependency1Node.ID())
+	assert.MapContainsKey(t, dependency1Node.ID(), readyNodes)
 	assert.NoError(t, dependency1Node.ResolveNode(dependencyNodeResolution))
-	assert.Equals(t, dependency1Node.ResolutionStatus(), dependencyNodeResolution)
 	readyNodes = d.PopReadyNodes()
 	assert.Equals(t, len(readyNodes), 1)
-	assert.Equals(t, readyNodes[0].ID(), dependentNode.ID())
-	assert.Equals(t, dependentNode.ResolutionStatus(), expectedDependentNodeResolution)
+	assert.MapContainsKey(t, dependentNode.ID(), readyNodes)
```

Comment:
The assertion at old line 169 shouldn't be deleted -- it should now be something similar to
```go
	assert.Equals(t, readyNodes[dependentNode.ID()], expectedDependentNodeResolution)
```
Right?

Reply:
Correct. Fixed.